static_src/test/functional/overview_quicklook.spec.js

```diff
@@ -43,7 +43,9 @@ describe('Overview page', function () {
     });
 
     it('is expanded', function () {
-      expect(quicklookElement.isExpanded()).toBe(true);
+      browser.waitUntil(function() {
+        return quicklookElement.isExpanded();
+      }, 2000, 'quicklook does not expand');
```

Comment:
webdriver.io is all sync anyway so you don't need to return a promise. Basically you'll move this `waitUntil` into a new `expand` method in `quicklook.element` and then on [line 42](https://github.com/18F/cg-dashboard/pull/998/commits/9d76f5fe0bb90dd06b7b95a5b51fdf2a51f2d27d#diff-1a9224665779b2b0186cf33b4f843d2dR42) you'll call `expand` instead of `click`. That click test block will handle the waiting. Does that make sense?